src/plugins/timelist/Timelist.vue

```diff
@@ -197,21 +197,22 @@ export default {
                 this.listActivities();
             }
         },
-        updateTimestamp(_bounds, isTick) {
+        updateTimestamp(bounds, isTick) {
             if (isTick === true && this.openmct.time.clock() !== undefined) {
                 this.updateTimeStampAndListActivities(this.openmct.time.clock().currentValue());
+            } else if (isTick === false && this.openmct.time.clock() === undefined) {
+                // set the start time for fixed time using the selected bounds start
+                this.updateTimeStampAndListActivities(bounds.start);
             }
         },
         setViewFromClock(newClock) {
             this.filterValue = this.domainObject.configuration.filter;
-            const isFixedTime = newClock === undefined;
-            if (isFixedTime) {
+            this.isFixedTime = newClock === undefined;
+            if (this.isFixedTime) {
                 this.hideAll = false;
                 this.showAll = true;
```

Comment:
We should not set `this.showAll` to true when it is fixed time span anymore. We want to honor the user's configuration of showing past/present/future which is done by calling `this.setViewBounds`.